docs/modules/ROOT/pages/objc.adoc

```diff
@@ -62,6 +62,39 @@ Cocoapods.
 pod install
 ----
 
+Starter code
+^^^^^^^^^^^^
+
+Open *ViewController.m* in Xcode and copy the following code in the `viewDidLoad` method. This snippet demonstrates
+how to run basic CRUD operations, a simple Query and running
+bi-directional replications with Sync Gateway.
+
+[source,objectivec]
+----
+include::{examplesdir}/SampleCodeTest.m[tags=getting-started]
+----
+
+Build and run. You should see the document ID and property printed to
+the console. The document was successfully persisted to the database.
+
+image:getting-started-ios.png[image]
+
+Before synchronizing documents to Sync Gateway you will need to disable
+App Transport Security. In the Xcode navigator, right-click on
```

Comment:
Is it more clear to also describe why the App Transport Security need to disable?